sessionctx/variable/session.go

```diff
@@ -742,7 +742,7 @@ type SessionVars struct {
 	PlannerSelectBlockAsName []ast.HintTable
 
 	// LockWaitTimeout is the duration waiting for pessimistic lock in milliseconds
-	// negative value means nowait, 0 means default behavior, others means actual wait time
+	// negative value means nowait, the default behavior is alwayswait, others means actual wait time
```

Comment:
```suggestion
	// negative value means nowait, others means actual wait time
```

Seems not accurate to me... Now it should not have "default behavior". The minimal value of the session var is 1 second.

I think it can be just removed.